_data/projects/bootcamp.yml

```diff
@@ -0,0 +1,15 @@
+---
+name: bootcamp
+desc: A mini social networking application using Flask
+site: https://github.com/SriNandan33/bootcamp
+tags:
+- oss
+- web
+- flask
+- python
+upforgrabs:
+  name: up-for-grabs
+  link: https://github.com/up-for-grabs/up-for-grabs.net/labels/up-for-grabs
```

Comment:
This link should point to a label in https://github.com/SriNandan33/bootcamp that's associated with issues which are ideal for new contributions. I see you have `Hacktoberfest` and `good first issue` in use - did you have a preference?

Reply:
Thanks you so much @shiftkey , I have updated changed the lable to Hacktoberfest and updated the link as well .

I have a question, should I create a new PR everytime I open an issue on my public repo?

Reply:
> I have a question, should I create a new PR everytime I open an issue on my public repo?

@SriNandan33 no need to do this - we have a GitHub Action that regenerates the stats for all projects daily. You can see an example of that in action here: #1385